src/orbitals.jl

```diff
@@ -53,8 +53,17 @@ function unsafe_unpack_ψ(x, sizes_ψ)
 end
 unpack_ψ(x, sizes_ψ) = deepcopy(unsafe_unpack_ψ(x, sizes_ψ))
 
-function random_orbitals(basis::PlaneWaveBasis{T}, kpt::Kpoint, howmany) where {T}
-    orbitals = similar(basis.G_vectors, Complex{T}, length(G_vectors(basis, kpt)), howmany)
-    randn!(TaskLocalRNG(), orbitals)  # Force the use of GPUArrays.jl's random function if using the GPU
-    ortho_qr(orbitals; array_type = array_type(basis))
+@static if VERSION < v"1.7"
```

Comment:
You can actually use this in the middle of a function. This saves you from duplicating 3 lines of code.

Reply:
Oh, neat! I underestimated the `@static` macro.